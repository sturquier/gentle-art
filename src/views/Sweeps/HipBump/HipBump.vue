<template>
  <h1>Hip bump sweep</h1>
</template>
