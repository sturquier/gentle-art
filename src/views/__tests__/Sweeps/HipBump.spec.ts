import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import HipBump from '../../Sweeps/HipBump/HipBump.vue'

describe('HipBump', () => {
  it('renders properly', () => {
    const wrapper = mount(HipBump)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
