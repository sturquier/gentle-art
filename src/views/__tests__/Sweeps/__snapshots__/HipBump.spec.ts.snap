// Vitest Snapshot v1

exports[`HipBump > renders properly 1`] = `"<h1>Hip bump sweep</h1>"`;
